### ISSUE TITLE
Support for path helpers

### ISSUE THREAD

**Issue by mr-feek** [psalm/psalm-plugin-laravel#59]
Currently, the likes of `base_path()` emits an `UnresolvableInclude`.

These are all of the path helpers to add support for 

```php
/**
     * Get the base path of the Laravel installation.
     *
     * @return string
     */
    public function basePath();

    /**
     * Get the path to the bootstrap directory.
     *
     * @param  string  $path Optionally, a path to append to the bootstrap path
     * @return string
     */
    public function bootstrapPath($path = '');

    /**
     * Get the path to the application configuration files.
     *
     * @param  string  $path Optionally, a path to append to the config path
     * @return string
     */
    public function configPath($path = '');

    /**
     * Get the path to the database directory.
     *
     * @param  string  $path Optionally, a path to append to the database path
     * @return string
     */
    public function databasePath($path = '');

    /**
     * Get the path to the environment file directory.
     *
     * @return string
     */
    public function environmentPath();

    /**
     * Get the path to the resources directory.
     *
     * @param  string  $path
     * @return string
     */
    public function resourcePath($path = '');

    /**
     * Get the path to the storage directory.
     *
     * @return string
     */
    public function storagePath();
```

**Comment by mr-feek** [comment]
Hey @muglug , any insight for how to inform psalm of these paths via the plugin?

**Comment by muglug** [comment]
A return type provider for that method that returns a literal string should work

**Comment by mr-feek** [comment]
Ah cool, i can probably just swap out the typehinted contract with the actual instance that provides these methods, thanks!

**Comment by mr-feek** [comment]
This should have been resolved in #99
